cmd/kube-proxy/app/server.go

```diff
@@ -690,6 +692,19 @@ func (s *ProxyServer) Run() error {
 	// functions must configure their shared informer event handlers first.
 	informerFactory.Start(wait.NeverStop)
 
+	// Make an informer that selects for our nodename.
+	informerFactory = informers.NewSharedInformerFactoryWithOptions(s.Client, s.ConfigSyncPeriod,
```

Comment:
Can we use a different variable name for this informer factory?

Reply:
Yes, will rename to `currentNodeInformerFactory`